src/main/java/com/arpnetworking/metrics/mad/model/DefaultRecord.java

```diff
@@ -181,19 +189,16 @@ public Builder setAnnotations(final ImmutableMap<String, String> value) {
             return this;
         }
 
-        // Called by OVal reflectively
-        @SuppressFBWarnings("UPM_UNCALLED_PRIVATE_METHOD")
-        private boolean validateAnnotations(final ImmutableMap<String, String> annotations) {
```

Comment:
Is the intent to no longer verify that the host, service and cluster exist in the Record?